spec.html

```diff
@@ -19962,13 +19975,11 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
               <pre><code class="javascript">constructor() {}</code></pre>
               using the syntactic grammar with the goal symbol |MethodDefinition[~Yield, ~Await]|.
         1. Set the running execution context's LexicalEnvironment to _classScope_.
-        1. Let _constructorInfo_ be DefineMethod of _constructor_ with arguments _proto_ and _constructorParent_
+        1. Let _constructorInfo_ be ! DefineMethod of _constructor_ with arguments _proto_, _className_ , and _constructorParent_.
```

Comment:
```suggestion
        1. Let _constructorInfo_ be ! DefineMethod of _constructor_ with arguments _proto_, _className_, and _constructorParent_.
```
(Delete space before comma.)